aaelf64-morello/aaelf64-morello.rst

```diff
@@ -0,0 +1,824 @@
+
+Trademark notice
+----------------
+
+The text of and illustrations in this document are licensed by Arm
+under a Creative Commons Attribution–Share Alike 4.0 International
+license ("CC-BY-SA-4.0”), with an additional clause on patents.
+The Arm trademarks featured here are registered trademarks or
+trademarks of Arm Limited (or its subsidiaries) in the US and/or
+elsewhere. All rights reserved. Please visit
+https://www.arm.com/company/policies/trademarks for more information
+about Arm’s trademarks.
+
+Copyright
+---------
+
+Copyright (c) |copyright-date|, Arm Limited and its affiliates.  All rights
+reserved.
+
+.. raw:: pdf
+
+   PageBreak
+
+.. contents::
+   :depth: 3
+
+.. raw:: pdf
+
+   PageBreak
+
+Preface
+=======
+
+Morello alpha
+-------------
+This document is an alpha proposal for Morello extensions to ELF for
+AArch64. Feedback welcome through your normal channels.
+
+
+Abstract
+--------
+
+This document describes the use of the Morello extensions to the ELF binary file
+format in the Application Binary Interface (ABI) for the Arm 64-bit architecture.
+
+
+Keywords
+--------
+
+ELF, AArch64 ELF, Morello, C64, ...
+
+References
+----------
+
+This document refers to, or is referred to by, the following documents.
+
+.. class:: aaelf64-morello-references
+
+.. table::
+
+  +------------------+-------------------------------------------------------------+----------------------------------------------------------------------------------+
+  | Ref              | External reference or URL                                   | Title                                                                            |
+  +==================+=============================================================+==================================================================================+
+  | MORELLO_AAELF64  | This document                                               | Morello extensions to ELF for the Arm 64-bit Architecture (AArch64).             |
+  +------------------+-------------------------------------------------------------+----------------------------------------------------------------------------------+
+  | AAELF64_         | IHI 0056                                                    | ELF for the Arm 64-bit Architecture (AArch64).                                   |
+  +------------------+-------------------------------------------------------------+----------------------------------------------------------------------------------+
+  | MORELLO_ARM_     | DDI0606                                                     | Arm® Architecture Reference Manual Supplement Morello for A-profile Architecture |
+  +------------------+-------------------------------------------------------------+----------------------------------------------------------------------------------+
+  | TLSDESC_         | http://www.fsfla.org/~lxoliva/writeups/TLS/paper-lk2006.pdf | TLS Descriptors for Arm. Original proposal document                              |
```

Comment:
Nit: could use replacement text [1] to reduce the width of this column.


[1] https://docutils.sourceforge.io/docs/ref/rst/directives.html#replacement-text

Reply:
Hi Francesco
Thanks for the suggestion.
I've tried to fix this in commit bd9fdb676f29a3ed7d9dc7e7995e0b28c82d5f33